packages/kbn-securitysolution-list-utils/src/helpers/index.ts

```diff
@@ -1030,16 +1030,12 @@ export const getMappingConflictsInfo = (field: DataViewField): FieldConflictsInf
  * Given an exceptions list, determine if any entries have an "IS" operator with a wildcard value
  */
 export const getHasWrongOperator = (items: ExceptionsBuilderReturnExceptionItem[]): boolean => {
-  let hasWrongOperator = false;
-  items[0]?.entries.forEach((e) => {
-    if (
+  return items[0]?.entries.some((e) => {
+    return (
       validateHasWildcardWithWrongOperator({
         operator: (e as ExceptionsBuilderReturnExceptionItem[number]).type,
         value: (e as ExceptionsBuilderReturnExceptionItem[number]).value,
-      })
-    ) {
-      hasWrongOperator = true;
-    }
+      }) === true
```

Comment:
Thanks for the change 👍 I think you do not need to compare this to === true, since validateHasWildcardWithWrongOperator already returns a boolean. 

What do you think about the below: 
```
export const getHasWrongOperator = (items: ExceptionsBuilderReturnExceptionItem[]): boolean => {
  return items[0]?.entries.some((e) =>
    validateHasWildcardWithWrongOperator({
      operator: (e as ExceptionsBuilderReturnExceptionItem[number]).type,
      value: (e as ExceptionsBuilderReturnExceptionItem[number]).value,
    })
  );
};
```